server/internal/platform/pg/user.go

```diff
@@ -0,0 +1,7 @@
+
+package pg
+
+// TODO: REMOVE STRUCT FROM HERE, AND DEFINE IN SEPERATE FILE IN FLOWTHRU FOLDER
+type User struct {
+	ID int
+}
```

Comment:
This should be moved into the flowthru directory and imported for use

Reply:
I believe there is already a user.go in the flowthru directory.  Can probably just delete this file.

Reply:
Thats true, can get rid of it. Additionally, user.go shouldn't be importing pg

Reply:
Isn't a user.go file still necessary since we will be splitting up how we handle the domain logic of each entity?
Also, what do you mean it shouldn't be named 'pg'?

Reply:
As of right now to keep things simple, I think it will be best to keep all models and entities within the `flowthru` package. So to assemble a user, the `pg` package will import a user struct from the `flowthru` package

Reply:
Ah I see, sounds good.